keras_cv/ops/point_cloud.py

```diff
@@ -329,3 +329,42 @@ def spherical_coordinate_transform(points):
     # Note: tf.atan2 takes in (y, x).
     phi = tf.atan2(points[..., 1], points[..., 0])
     return tf.stack([dist, theta, phi], axis=-1)
+
+
+def within_a_frustum(points, center, r_distance, theta_width, phi_width):
+    """Check if 3d points are within a 3d frustum.
```

Comment:
Maybe specify in the docstring that this is a conal frustum? (At least that's what I am guessing from the code)

Reply:
Done.